app/views/profile/orders/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Order Total for <%= current_user.name %></h1>
+
+<% current_user.orders.each do |order| %>
+<ul>
+  <% order.order_items.each do |order_item| %>
+  <br>
+  <li>Order ID: <%= order_item.id %> </li>
+  <li>Order Created At: <%= order_item.created_at %> </li>
+  <li>Order Updated At: <%= order_item.updated_at %> </li>
```

Comment:
What I would do here is add `Item` after each order to specify that the information corresponds to an item on the order not the order itself.

Also not sure if this is a part of the card but displaying the `id` of the order item doesn't seem neccessary.